ark/utils/deepcell_service_utils.py

```diff
@@ -136,8 +141,12 @@ def zip_write(zip_path):
 
         with ZipFile(zip_files[-1], "r") as zipObj:
             for name in zipObj.namelist():
-                with open(os.path.join(deepcell_output_dir, name), mode='wb') as f:
-                    f.write(zipObj.read(name))
+                mask_path = os.path.join(deepcell_output_dir, name)
```

Comment:
Did you actually run the notebook with real data to make sure this works as expected, as well as inspecting the segmentation mask output from before and after the change to make sure segmentation looks identical?